Pipfile

```diff
@@ -68,3 +68,7 @@ types-pyyaml = ">=6.0"
 types-requests = ">=2.31"
 types-toml = ">=0.10"
 wheel = "==0.41.0"
+pytest-bdd = "==6.1.1"
+playwright = "==1.40.0"
+hac-playwright = { file = "http://h2o-public-test-data.s3.amazonaws.com/e2e-testing/hac_playwright-1.40.0-py3-none-any.whl" }
```

Comment:
[This ](https://github.com/h2oai/h2o-llmstudio/pull/561/files#diff-230078d672f10d17463a8a6265cad825b790885898256a3365be90685caac58d) is currently blocking using 1.40

Reply:
okay, installation the previous available version 1.38.0 instead